qcodes/instrument_drivers/Minicircuits/USBHIDMixin.py

```diff
@@ -61,68 +58,83 @@ def __init__(self, name, instance_id: str=None, timeout: float=2,
 
         self._device = devs[0]
         self._device.open()
+
         self._data_buffer: Optional[bytes] = None
-        self._timeout = timeout
         self._device.set_raw_data_handler(self._handler)
 
+        self._timeout = timeout
+        self._tries_per_second = 5
+
         super().__init__(name, **kwargs)
 
-    def _handler(self, data: bytes) ->None:
+    def _handler(self, data: bytes) -> None:
         self._data_buffer = data
 
     def _get_data_buffer(self)->Optional[bytes]:
         data = self._data_buffer
         self._data_buffer = None
         return data
 
-    def _pack_string(self, cmd: str) ->bytes:
+    def _pack_string(self, cmd: str) -> bytes:
         raise NotImplementedError("Please subclass")
 
-    def _unpack_string(self, response: bytes) ->str:
+    def _unpack_string(self, response: bytes) -> str:
         raise NotImplementedError("Please subclass")
 
-    def write_raw(self, cmd: str) ->None:
+    def write_raw(self, cmd: str) -> None:
         """
-        Send binary data to the human interface device
+        Send a string command to the human interface device
+
+        The given command is processed by `_pack_string` method to return a
```

Comment:
I don't think this belongs in a docstring. Only put stuff in here which gives info on how to call a function, what it returns and what it does. The user does not care that the command is processed by a private method called `_pack_string`

Reply:
in this particular case, the end user of the drivers will not ever directly work with `write_raw`. However, a person who might be working on debugging the driver or adding features to it will look at `write_raw` and he/she will be probably glad to hear this extra info about `_pack_string` (although it is sort of clear from the code).